qpc/scan/utils.py

```diff
@@ -95,12 +95,41 @@ def _get_optional_products(disabled_optional_products):
     return optional_product_status
 
 
+def get_extended_products(enabled_extended_product_search,
+                          ext_product_search_dirs, edit):
+    """Construct a dictionary based on the enabled extended product search args.
+
+    :param enabled_extended_product_search The products to enable
+    :param ext_product_search_dirs: The search dirs for the extended products
+    :param edit: boolean regarding if we are editing the extended products
+    :returns: a dictionary representing the enabled search status of extended
+    products
+    """
+    enabled_product_search_status = {}
+
+    if enabled_extended_product_search:
+        for product in enabled_extended_product_search:
+            enabled_product_search_status[product] = True
+    else:
+        if not edit:
+            return None
+    # add the search directories if they are provided
+    if ext_product_search_dirs:
+        enabled_product_search_status['search_directories'] \
+            = ext_product_search_dirs
+
+    return enabled_product_search_status
```

Comment:
why are we calling it `enabled_product_search_status`? It doesn't feel like a status.